libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.cpp

```diff
@@ -27,7 +27,7 @@
 
 #ifdef HTTPCLIENT_1_1_COMPATIBLE
 #include <ESP8266WiFi.h>
-#include <WiFiClientSecure.h>
+#include <WiFiClientSecureAxTLS.h>
```

Comment:
I'd have done these changes differently, but I guess this is OK too and much less work.  Since axTLS is going away anyway, fine by me.

The "proper" way to do this, I believe, would be to change `WiFiClientSecure.h` to `using BearSSL`, then no changes to headers in any other files.

There would be changes to all object instantiations from `WiFiClientSecure` to `axTLS::WiFiClientSecure` in the examples, and then removal of the `BearSSL::` prefix everywhere.